packages/database/src/migrations/20211101040146-WAI-1021-TongaContactTracingVisual-modifies-data.js

```diff
@@ -59,16 +82,25 @@ const REPORT_CONFIG = {
       groupBy: ['organisationUnit', 'period'],
       transform: 'mergeRows',
     },
+    {
+      insert: {
+        isCase: '=exists($TO_C19CRF112)',
+        caseSymptomatic: `=any(${CASE_SYMPTOMS.map(x => `eq($${x}, 'Yes')`).join(
+          ',',
+        )}) ? 'Yes' : 'No'`,
+      },
+      transform: 'updateColumns',
+    },
     {
       insert: {
         Case:
-          "=exists($TO_C19CRF112) ? $TO_C19CRF112 : exists($TO_C19CLF02) ? orgUnitIdToCode($TO_C19CLF02) : 'Case not found'",
-        Status: '=$TO_FCF_05',
-        Contacts: '=exists($TO_C19CLF22) ? $TO_C19CLF22 : $organisationUnit',
-        Symptomatic: "=translate($TO_FCF_06, {'0': 'No', '1': 'Yes'})",
+          "=$isCase ? $TO_C19CRF112 : exists($TO_C19CLF02) ? orgUnitIdToCode($TO_C19CLF02) : 'Case not found'",
+        Status: '=$isCase ? $TO_C19CRF12 : $TO_FCF_05',
+        Contacts: '=$isCase ? $organisationUnit : $TO_C19CLF22',
+        Symptomatic: "=$isCase ? $caseSymptomatic : translate($TO_FCF_06, {'0': 'No', '1': 'Yes'})",
         'Phone Number': '=$TO_C19CLF10',
         'Last Contacted': '=periodToDisplayString($period)',
-        'Vaccination Status': '=$TO_C19CLF26',
+        'Vaccination Status': '=$isCase ? $TO_C19CRF85 : $TO_C19CLF26',
         'Last Contact with Case':
           '=exists($TO_C19CLF16) ? periodToDisplayString(dateStringToPeriod($TO_C19CLF16)) : undefined',
       },
```

Comment:
Almost missed the little `exclude: '*'` below this because it's hidden in the github preview, but that's very handy to get rid of the intermediary helper variables